### DISCUSSION TITLE
Easy way of exporting current working FS

### DISCUSSION THREAD

**Discussion by Clay-CL** [Joshua-Riek/ubuntu-rockchip#554] [Q&A]
I've intsalled the v1.31 release of the ubuntu 22 server for my orange pi 5B.
I've customised the filesystem, installed some libraries and added couple of config files.
I wanted to take a backup of this image and load it into the eMMC of another orange PI.
Is there any clean way of doing this?

Your answers are much appreciated

**Comment by Clay-CL** [comment]
@Joshua-Riek any thoughts on this?

**Comment by Joshua-Riek** [comment]
use the command ubuntu-rockchip-install to copy your currently running system to another device. But depending on your modifications it may break. Or you can use a full copy with dd.
```
sudo ubuntu-rockchip-install /dev/mmcblk0
```

**Reply by Clay-CL** [reply]
Hi Joshua, thanks for the prompt reply. This would work, but is there any way of exporting it as an image file?

**Reply by Joshua-Riek** [reply]
You could use dd for this, but I do not know the exact command on the top of my head.

**Reply by Eternalightwithin** [reply]
It would be something like "dd if=/dev/sdb1 of=/backup/sdb1.img" to backup and then reverse it with dd if=/backup/sdb1.img of=/dev/sdb2 to restore.  I did this when I brought a RPi CM4 and was learning Linux (Rasbian and Ubuntu).  Lots of reflashing when I would brick the install.

I couldn't remember the syntax exactly so I pulled this from https://tecadmin.net/drive-and-partition-backup-examples-with-dd-command/
